scripts/lib-node.sh

```diff
@@ -39,6 +43,6 @@ function nodeargs () {
         printf -- "--pbft-signature-threshold 0.7 "
         printf -- "--require-network-magic "
         printf -- "--database-path db "
-        printf -- "--topology configuration/simple-topology.json "
+        topology
```

Comment:
Erm. Ok, nice, but not really sure if this is actually doing something?

Reply:
@ksaric For some reason the script was not registering the `--topology` argument. This change fixed that for some reason :man_shrugging:.

Reply:
@ksaric You're right, I reverted and it still works. Strange.